webapp/TargetedMS/js/QCPlotHelperBase.js

```diff
@@ -602,6 +610,12 @@ Ext4.define("LABKEY.targetedms.QCPlotHelperBase", {
         if (plotType !== LABKEY.vis.TrendingLinePlotType.MovingRange && plotType !== LABKEY.vis.TrendingLinePlotType.LeveyJennings) {
             yScaleLabel = 'Sum of Deviations'
         }
+        if (plotType === LABKEY.vis.TrendingLinePlotType.TrailingMean) {
```

Comment:
Is this check needed? I think the others are just when we're overriding the label based on the data transformation. Not a big deal either way though